app/bundles/ReportBundle/Model/ReportModel.php

```diff
@@ -363,55 +363,32 @@ public function getGraphList($context = 'all')
      *
      * @throws \Exception
      */
-    public function exportResults($format, $report, $reportData)
+public function exportResults($format, $report, $reportData, $handle, $page)
```

Comment:
This line should be indented by 4 spaces to the right. A problem is that by adding these additional mandatory parameters you broke BC. The method can be used in 3rd party plugins and this will break them.